src/services/sync/strategies/consumerActions/UserAction.js

```diff
@@ -127,6 +127,9 @@ class UserAction extends BaseConsumerAction {
 		if (user.ldapDn !== foundUser.ldapDn) {
 			updateObject.ldapDn = user.ldapDn;
 		}
+
+		updateObject.lastSyncedAt = new Date();
```

Comment:
the logic is a bit hidden now, so the next person looking at it will have a harder time understanding
I would move into updateUserAndAccount, and maybe adding a comment